src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_pending_resumption_handler.cc

```diff
@@ -220,6 +226,19 @@ bool RCPendingResumptionHandler::IsPendingForResponse(
   return it != waiting_for_response_modules_.end();
 }
 
+bool RCPendingResumptionHandler::IsAnotherAppsSubscribedOnTheSameModule(
+    const ModuleUid& module_uid) const {
+  const auto& it = subscribed_modules.find(module_uid);
```

Comment:
We are doing subscriptions not only via RCPendingResumptionHandler.
subscribed_modules it is only subscribed via resumption. But at the same time, regular subscriptions are possible. 

Please reimplement IsAnotherAppsSubscribedOnTheSameModule to check the status subscription of the apps but not only resumed one

Reply:
@LuxoftAKutsan 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/119/commits/e8ebdba4bca068592d98be4cdcee87e4b608ae29